src/desktop/components/split_test/running_tests.coffee

```diff
@@ -45,4 +45,13 @@ module.exports = {
     weighting: 'equal'
     edge: 0
     control_group: 9
+  new_collect_page:
+    key: 'new_collect_page',
```

Comment:
Maybe this wants to be commented out as well until we start the test, these cookies will be set even if they are not used.

Reply:
Right, it's ok that the group bucket cookie gets set. It won't skew our test results as the impression tracking is disabled, which is what matters.